content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -153,7 +153,7 @@ secretGenerator:
 - name: example-secret-2
   literals:
   - username=admin
-  - password=secert 
+  - password=secert
```

Comment:
aside: I suspect this is supposed to read “secret”.

Reply:
@sftim This is a nice suggestion.
I fixed the typo and the following command's output.